src/cnxml/se.tsx

```diff
@@ -440,12 +440,12 @@ function link(node: Link, attrs: CommonAttrs, children: JSX.Node): JSX.Node {
 
 function list(node: List, attrs: CommonAttrs, children: JSX.Node): JSX.Node {
     function mapChild(child: JSX.Node): JSX.Node {
-        if (child == null || typeof child === 'string' || child instanceof globalThis.Node) {
+        if (Array.isArray(child)) return child.map(mapChild)
+
+        if (JSX.nodeIsNotJSXElement(child)) {
             return child
         }
 
-        if (Array.isArray(child)) return child.map(mapChild)
-
         if ((child.name.namespace == null || child.name.namespace === JSX.CNXML_NAMESPACE)
         && child.name.local === 'list') {
             return <item xmlns={JSX.CNXML_NAMESPACE}>{child}</item>
```

Comment:
This change is unrelated to the PR.

Reply:
https://github.com/openstax-poland/cnx-designer/pull/117#discussion_r571933375